tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/retry/clientserver/RetryClientForZeroJitter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2016-2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.retry.clientserver;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.sql.Connection;
+
+import org.eclipse.microprofile.faulttolerance.Retry;
+
+/**
+ * A client to check for proper processing of jitter = 0 on @Retry
+ * 
+ * @author <a href="mailto:doychin@dsoft-bg.com">Doychin Bondzhev</a>
+ *
+ */
+@ApplicationScoped
+public class RetryClientForZeroJitter {
+
+    private long totalRetryTime = 0;
+
+    private long previousTime = 0;
+
+    private int retries = -1; // first call is normal call
+
+    @Retry(jitter = 0)
```

Comment:
Hi @doychin can you please set the method to retry at least 3 times?
The 2 first retries should fail and the second should pass.
You can push the changes to this branch and the PR will be automatically updated, no need to open a new PR. 
Thanks!

Reply:
isn't maxRetries by default 3? Anyway I will add it as parameter to the annotation. So I add check and on the third retry it will just return without exception. The test code will change to consider exception as fail and when there is no exception it will check for number of retries and total time to execute.

Reply:
yes, the default is 3, but the test should be explicit and resilient to defaults change.